server/vehicle/main.lua

```diff
@@ -204,5 +206,24 @@ function Ox.GeneratePlate()
     end
 end
 
+---Creates a unique vehicle vin number.
+---@return string
+function Ox.GenerateVin()
+    local vin = table.create(17, 0)
+    while true do
+        ::start::
+        for i = 1, 17 do
+            vin[i] = math.random(0, 1) == 1 and string.char(math.random(65, 90)) or math.random(0, 9)
+        end
+
+        local str = table.concat(vin)
+        if db.isVinAvailable(str) then
+            return str
+        end
```

Comment:
Formatting. Line break between variable assignment and if statement; one-liner return when simple (just match plate generation).